package.json

```diff
@@ -37,14 +37,17 @@
     "start:playground": "npm run storybook",
     "std-version": "standard-version -m \"chore(release): version %s build ${TRAVIS_BUILD_NUMBER} [ci skip]\"",
     "storybook": "start-storybook -p 12123",
+    "storybook:ci": "start-storybook -p 12123 --ci --quiet",
```

Comment:
The `--ci` flag stops this from launching storybook.

Reply:
To avoid having too many duplicate commands, I believe we could rewrite this as:
```
"storybook:ci": "npm run storybook -- --ci --quiet"
```